modules/sno-du-configuring-logging-locally-and-forwarding.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-logging-locally-and-forwarding_{context}"]
+= Configuring logging locally and forwarding
```

Comment:
Do you need a prerequisite that Cluster Logging must be installed?

Reply:
No.

Reply:
@yliu127 @imiller0 Please confirm.

Reply:
The ClusterLogging configuration does require that the Cluster Logging operator be installed. The Subscriptions section of this document includes the Cluster Logging operator so the pre-requisite is, I believe, covered by that.